{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# *Acropora palmata* gene expression data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This workflow downloads and processes *A. palmata* [gene expression data](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36983) and joins it with CpG O/E data for subsequent analysis in R. All samples are included in a single file, but the file content and contig names had to be edited prior to analysis in R."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[Errno 2] No such file or directory: '../analyses/Apalm'\n",
      "/Users/jd/Documents/Projects/Coral-CpG-ratio-MS/analyses/Apalm\n"
     ]
    }
   ],
   "source": [
    "cd ../analyses/Apalm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 26.1M  100 26.1M    0     0  4498k      0  0:00:05  0:00:05 --:--:-- 5465k\n"
     ]
    }
   ],
   "source": [
    "!curl -O ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE36nnn/GSE36983/matrix/GSE36983_series_matrix.txt.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!gzip -d GSE36983_series_matrix.txt.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "!Series_title\t\"Variation in the transcriptional response of threatened coral larvae to elevated temperatures\"\r\n",
      "!Series_geo_accession\t\"GSE36983\"\r\n",
      "!Series_status\t\"Public on Dec 13 2012\"\r\n",
      "!Series_submission_date\t\"Apr 02 2012\"\r\n",
      "!Series_last_update_date\t\"Mar 14 2013\"\r\n",
      "!Series_pubmed_id\t\"23331636\"\r\n",
      "!Series_summary\t\"Coral populations have declined worldwide largely due to increased sea surface temperatures. Recovery of coral populations depends in part upon larval recruitment. Many corals reproduce during the warmest time of year when further increases in temperature can lead to low fertilization rates of eggs and high larval mortality. Microarray experiments were designed to capture and assess variability in the thermal stress responses of Acropora palmata larvae from Puerto Rico. Transcription profiles showed a striking acceleration of normal developmental gene expression patterns with increased temperature. The transcriptional response to heat suggested rapid depletion of larval energy stores via peroxisomal lipid oxidation and included key enzymes that indicated the activation of the glyoxylate cycle. High temperature also resulted in expression differences in key developmental signalling genes including the conserved WNT pathway that is critical for pattern formation and tissue differentiation in developing embryos. Expression of these and other important developmental and thermal stress genes such as ferritin, heat shock proteins, cytoskeletal components, cell adhesion and autophagy proteins also varied among larvae derived from different parent colonies. Disruption of normal developmental and metabolic processes will have negative impacts on larval survival and dispersal as temperatures rise. However, it appears that variation in larval response to high temperature remains despite the dramatic population declines. Further research is needed to determine whether this variation is heritable or attributable to maternal effects.\"\r\n",
      "!Series_overall_design\t\"Hybridization followed a dual channel loop design using two biological replicates (dye-swapped) from each treatment that maximized power to detect differential expression in contrasts between temperatures and batches (within time-points) as well as the amount of data obtained from each slide (Simon and Dobbin 2003). A total of 18 arrays on two 12 plex slides were used (Table S1). Additional samples from other sub-batches (b3/b4) were included in the microarray experiment but are not used in this analysis. Each array measures the expression level of 135,185 genes from the elkhorn coral (Acropora palmata) transcriptome (Polato et al. 2011). Two 60-mer probes were designed for each contig (n = 85,260), and a single probe was designed for each singleton sequence (n = 45,390). Two additional probes each were developed for sequences associated with annotation information relating to calcium metabolism and stress response (n = 4,798). Replicate probes for individual sequences from the assembled transcriptome were not identical; rather they represented multiple different 60-mer sequences from the original template.\"\r\n",
      "!Series_type\t\"Expression profiling by array\"\r\n",
      "!Series_contributor\t\"Nicholas,R,Polato\"\r\n",
      "!Series_contributor\t\"Iliana,B,Baums\"\r\n",
      "!Series_contributor\t\"Naomi,S,Altman\"\r\n",
      "!Series_sample_id\t\"GSM907976 GSM907977 GSM907978 GSM907979 GSM907980 GSM907981 GSM907982 GSM907983 GSM907984 GSM907985 GSM907986 GSM907987 GSM907988 GSM907989 GSM907990 GSM907991 GSM907992 GSM907993 GSM907994 GSM907995 GSM907996 GSM907997 GSM907998 GSM907999 GSM908000 GSM908001 GSM908002 GSM908003 GSM908004 GSM908005 GSM908006 GSM908007 GSM908008 GSM908009 GSM908010 GSM908011 \"\r\n",
      "!Series_contact_name\t\"Nicholas,,Polato\"\r\n",
      "!Series_contact_email\t\"polato@psu.edu\"\r\n",
      "!Series_contact_laboratory\t\"Baums\"\r\n",
      "!Series_contact_department\t\"Biology\"\r\n",
      "!Series_contact_institute\t\"Penn State University\"\r\n",
      "!Series_contact_address\t\"208 Mueller Lab\"\r\n",
      "!Series_contact_city\t\"University Park\"\r\n",
      "!Series_contact_state\t\"PA\"\r\n",
      "!Series_contact_zip/postal_code\t\"16802\"\r\n",
      "!Series_contact_country\t\"USA\"\r\n",
      "!Series_supplementary_file\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/series/GSE36983/GSE36983_RAW.tar\"\r\n",
      "!Series_platform_id\t\"GPL15393\"\r\n",
      "!Series_platform_taxid\t\"6131\"\r\n",
      "!Series_sample_taxid\t\"6131\"\r\n",
      "!Series_relation\t\"BioProject: http://www.ncbi.nlm.nih.gov/bioproject/PRJNA157441\"\r\n",
      "\r\n",
      "!Sample_title\t\"RP576-B3-24H-29C\"\t\"RP1293-B2-24H-27C\"\t\"RP555-B4-24H-27C\"\t\"RP568-B3-24H-29C\"\t\"RP573-B5-24H-29C\"\t\"RP561-B4-24H-27C\"\t\"RP557-B3-24H-27C\"\t\"RP566-B5-24H-29C\"\t\"EP606-B2-24H-29C\"\t\"RP564-B3-24H-27C\"\t\"RP553-B5-24H-27C\"\t\"EP602-B2-24H-29C\"\t\"RP574-B4-24H-29C\"\t\"RP560-B5-24H-27C\"\t\"RP1295-B2-24H-27C\"\t\"RP567-B4-24H-29C\"\t\"RP1161-B2-72H-29C\"\t\"RP1156-B2-72H-27C\"\t\"RP1157-B2-72H-27C\"\t\"RP1162-B2-72H-29C\"\t\"RP1182-B3-48H-29C\"\t\"RP1177-B5-48H-27C-Cy5\"\t\"RP1169-B4-48H-27C\"\t\"RP1190-B3-48H-29C\"\t\"RP1189-B5-48H-29C\"\t\"RP1175-B4-48H-27C\"\t\"RP1168-B3-48H-27C\"\t\"RP1187-B5-48H-29C\"\t\"RP1103-B2-48H-29C\"\t\"RP1178-B3-48H-27C\"\t\"RP1177-B5-48H-27C-Cy3\"\t\"RP1102-B2-48H-29C\"\t\"RP1188-B4-48H-29C\"\t\"RP1174-B5-48H-27C\"\t\"RP597-B2-48H-27C\"\t\"RP1186-B4-48H-29C\"\r\n",
      "!Sample_geo_accession\t\"GSM907976\"\t\"GSM907977\"\t\"GSM907978\"\t\"GSM907979\"\t\"GSM907980\"\t\"GSM907981\"\t\"GSM907982\"\t\"GSM907983\"\t\"GSM907984\"\t\"GSM907985\"\t\"GSM907986\"\t\"GSM907987\"\t\"GSM907988\"\t\"GSM907989\"\t\"GSM907990\"\t\"GSM907991\"\t\"GSM907992\"\t\"GSM907993\"\t\"GSM907994\"\t\"GSM907995\"\t\"GSM907996\"\t\"GSM907997\"\t\"GSM907998\"\t\"GSM907999\"\t\"GSM908000\"\t\"GSM908001\"\t\"GSM908002\"\t\"GSM908003\"\t\"GSM908004\"\t\"GSM908005\"\t\"GSM908006\"\t\"GSM908007\"\t\"GSM908008\"\t\"GSM908009\"\t\"GSM908010\"\t\"GSM908011\"\r\n",
      "!Sample_status\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\t\"Public on Dec 13 2012\"\r\n",
      "!Sample_submission_date\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\t\"Apr 02 2012\"\r\n",
      "!Sample_last_update_date\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\t\"Dec 13 2012\"\r\n",
      "!Sample_type\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\t\"RNA\"\r\n",
      "!Sample_channel_count\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\t\"1\"\r\n",
      "!Sample_source_name_ch1\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\t\"Acropora palmata larval culture\"\r\n",
      "!Sample_organism_ch1\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\t\"Acropora palmata\"\r\n",
      "!Sample_characteristics_ch1\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\t\"tissue type: larval culture\"\r\n",
      "!Sample_characteristics_ch1\t\"batch: batch 3\"\t\"batch: batch 2\"\t\"batch: batch 4\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 4\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 2\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 2\"\t\"batch: batch 4\"\t\"batch: batch 5\"\t\"batch: batch 2\"\t\"batch: batch 4\"\t\"batch: batch 2\"\t\"batch: batch 2\"\t\"batch: batch 2\"\t\"batch: batch 2\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 4\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 4\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 2\"\t\"batch: batch 3\"\t\"batch: batch 5\"\t\"batch: batch 2\"\t\"batch: batch 4\"\t\"batch: batch 5\"\t\"batch: batch 2\"\t\"batch: batch 4\"\r\n",
      "!Sample_characteristics_ch1\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 24 hours\"\t\"age: 72 hours\"\t\"age: 72 hours\"\t\"age: 72 hours\"\t\"age: 72 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\t\"age: 48 hours\"\r\n",
      "!Sample_characteristics_ch1\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\t\"temperature: 29°C\"\t\"temperature: 27°C\"\t\"temperature: 27°C\"\t\"temperature: 29°C\"\r\n",
      "!Sample_treatment_protocol_ch1\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\t\"Larvae were reared at a control temperature (27°C) or elevated temperature (29°C). Replicates of each batch culture (n = 2) were kept in separate 1L plastic containers (with mesh sides to allow for water exchange). Plastic 1 L containers (n = 10) were suspended in replicate 45L polycarbonate bins (n = 4) at each temperature (n = 2). Water in each bin was circulated with an aquarium pump and changed daily with filtered sea water preheated to the target temperature. Temperatures were maintained within ±1°C with ¼ hp aquarium chillers (Current USA, CA) and monitored with HOBO data loggers (Onset Co., MA).\"\r\n",
      "!Sample_growth_protocol_ch1\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\t\"Gametes from multiple parent colonies were collected during the 2009 mass spawning event on the reef off of Rincon, Puerto Rico. The genotypic identity of each colony targeted for gamete collection was determined using microsatellite markers (Baums, Hughes et al. 2005). Batch cultures, consisting of four genotypically distinct parents each, were generated by combining sperm and eggs from selected parent colonies. After fertilization (one hour), the zygotes were rinsed with filtered sea water and transferred to temperature controlled aquaria.\"\r\n",
      "!Sample_molecule_ch1\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\t\"total RNA\"\r\n",
      "!Sample_extract_protocol_ch1\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\t\"Embryos for the microarray experiment were sampled at 24, 48, and 72 hours (only samples from batch 2 were available at 72h) and preserved in RNAlater (Ambion, TX) followed by storage at -80°C. Total RNA was extracted from approximately 30-100 larvae from each sample using the RNeasy Mini Kit (Qiagen, CA). Quality and concentration of total RNA was as-sessed on an Agilent 2100 Bioanalyzer to ensure that high molecular weight RNA was present.\"\r\n",
      "!Sample_label_ch1\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\t\"Cy3\"\t\"Cy5\"\r\n",
      "!Sample_label_protocol_ch1\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\t\"To prepare samples for microarray hybridization, one cycle of amplification was per-formed on 1ug of each RNA sample using the Amino Allyl MessageAmp II aRNA Amplification Kit (Ambion Life Technologies, AM1753) following the manufacturer’s protocol.  Dye coupling of 15 ug of aRNA was performed with Cy3 or Cy5 (GE Health Care, RPN5661), and subsequently purified according to the Ambion Kit instructions. For each pair of samples that were to be hy-bridized to the same array, 2µg each of the Cy3 and Cy5 labeled sample were combined and fragmented using RNA Fragmentation Reagents (Ambion, AM8740) according the manufactur-er’s instructions, then dried down completely in a speed-vac.\"\r\n",
      "!Sample_taxid_ch1\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\t\"6131\"\r\n",
      "!Sample_hyb_protocol\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\t\"Samples were resuspended in the appropriate tracking control and hybridization solution master mix was added following manufacturer’s instructions (Roche NimbleGen). Following two 5-minute incubations at 95°C and 42°C, the mixer was attached to the array and hybridization solutions were added according to the manufacturer’s instructions. Hybridization was performed while mixing overnight at 42°C in a MAUI hybridization instrument (BioMicro Systems, UT). Hybridized slides were washed according to the manufacturer’s instructions (Roche NimbleGen), and spun at 1000 RPM for 3 minutes in a 50 ml conical tube filled with N2 gas. Dried slides were placed in fresh tubes with 2.5 ml of DyeSaver (Genisphere Inc.) and rotated for 45 seconds to coat the slide. A final spin at 700 RPM for 3 seconds removed excess DyeSaver.\"\r\n",
      "!Sample_scan_protocol\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\t\"Scanning was performed with an Axon GenePix 4000B\"\r\n",
      "!Sample_description\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"Used a Cy3 replicate of sample RP1177 because a batch 2 sample for this condition was not available. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"Used a Cy5 replicate of sample RP1177. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\t\"This sample was not used for analysis after initial normalization. A batch extract from 30-100 coral larvae derived from the same set of four parent colonies from Rincon, Puerto Rico, and raised at a common temperature for a specified time since fertilization.\"\r\n",
      "!Sample_data_processing\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\t\"Raw probe intensities were read into R for analysis in the Bioconductor package LIMMA (Smyth 2005; R_Development_Core_Team 2008). Probe intensities were normalized within arrays using lowess normalization, and between arrays using the Aquantile normalization method which ensures that the average intensities have the same empirical distribution across arrays (Yang and Thorne 2003).\"\r\n",
      "!Sample_platform_id\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\t\"GPL15393\"\r\n",
      "!Sample_contact_name\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\t\"Nicholas,,Polato\"\r\n",
      "!Sample_contact_email\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\t\"polato@psu.edu\"\r\n",
      "!Sample_contact_laboratory\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\t\"Baums\"\r\n",
      "!Sample_contact_department\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\t\"Biology\"\r\n",
      "!Sample_contact_institute\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\t\"Penn State University\"\r\n",
      "!Sample_contact_address\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\t\"208 Mueller Lab\"\r\n",
      "!Sample_contact_city\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\t\"University Park\"\r\n",
      "!Sample_contact_state\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\t\"PA\"\r\n",
      "!Sample_contact_zip/postal_code\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\t\"16802\"\r\n",
      "!Sample_contact_country\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\t\"USA\"\r\n",
      "!Sample_supplementary_file\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907976/GSM907976_3042011_466388A01_RP576_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907977/GSM907977_3042011_466388A01_RP1293_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907978/GSM907978_3042011_466388A02_RP555_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907979/GSM907979_3042011_466388A02_RP568_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907980/GSM907980_3042011_466388A03_RP573_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907981/GSM907981_3042011_466388A03_RP561_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907982/GSM907982_3042011_466388A04_RP557_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907983/GSM907983_3042011_466388A04_RP566_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907984/GSM907984_3042011_466388A05_EP606_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907985/GSM907985_3042011_466388A05_RP564_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907986/GSM907986_3042011_466388A06_RP553_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907987/GSM907987_3042011_466388A06_EP602_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907988/GSM907988_3042011_466388A07_RP574_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907989/GSM907989_3042011_466388A07_RP560_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907990/GSM907990_3042011_466388A08_RP1295_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907991/GSM907991_3042011_466388A08_RP567_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907992/GSM907992_3042011_466388A09_RP1161_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907993/GSM907993_3042011_466388A09_RP1156_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907994/GSM907994_3042011_466388A10_RP1157_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907995/GSM907995_3042011_466388A10_RP1162_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907996/GSM907996_3022011_466433A01_RP1182_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907997/GSM907997_3022011_466433A01_RP1177_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907998/GSM907998_3022011_466433A02_RP1169_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM907nnn/GSM907999/GSM907999_3022011_466433A02_RP1190_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908000/GSM908000_3022011_466433A03_RP1189_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908001/GSM908001_3022011_466433A03_RP1175_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908002/GSM908002_3022011_466433A04_RP1168_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908003/GSM908003_3022011_466433A04_RP1187_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908004/GSM908004_3022011_466433A05_RP1103_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908005/GSM908005_3022011_466433A05_RP1178_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908006/GSM908006_3022011_466433A06_RP1177_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908007/GSM908007_3022011_466433A06_RP1102_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908008/GSM908008_3022011_466433A07_RP1188_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908009/GSM908009_3022011_466433A07_RP1174_635.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908010/GSM908010_3022011_466433A08_RP597_532.pair.gz\"\t\"ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM908nnn/GSM908011/GSM908011_3022011_466433A08_RP1186_635.pair.gz\"\r\n",
      "!Sample_data_row_count\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\t\"137604\"\r\n",
      "!series_matrix_table_begin\r\n",
      "\"ID_REF\"\t\"GSM907976\"\t\"GSM907977\"\t\"GSM907978\"\t\"GSM907979\"\t\"GSM907980\"\t\"GSM907981\"\t\"GSM907982\"\t\"GSM907983\"\t\"GSM907984\"\t\"GSM907985\"\t\"GSM907986\"\t\"GSM907987\"\t\"GSM907988\"\t\"GSM907989\"\t\"GSM907990\"\t\"GSM907991\"\t\"GSM907992\"\t\"GSM907993\"\t\"GSM907994\"\t\"GSM907995\"\t\"GSM907996\"\t\"GSM907997\"\t\"GSM907998\"\t\"GSM907999\"\t\"GSM908000\"\t\"GSM908001\"\t\"GSM908002\"\t\"GSM908003\"\t\"GSM908004\"\t\"GSM908005\"\t\"GSM908006\"\t\"GSM908007\"\t\"GSM908008\"\t\"GSM908009\"\t\"GSM908010\"\t\"GSM908011\"\r\n"
     ]
    }
   ],
   "source": [
    "!head -68 GSE36983_series_matrix.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\"AOKF1013_g2_cP00480\"\t8.752858025\t8.505150716\t9.472343652\t9.408601785\t9.482842255\t9.476583784\t9.927173819\t10.12591784\t10.21694941\t10.2803595\t9.670467134\t10.39275672\t8.7638457\t9.368024881\t8.716463095\t9.018597894\t9.823742505\t9.993787246\t9.133042104\t9.136289757\t9.611697945\t9.356880249\t9.649270508\t9.34780996\t9.496325421\t9.288494465\t9.988583534\t9.754569643\t9.510846045\t9.467298681\t9.366147084\t9.308251806\t9.317440431\t9.143020676\t9.7804244\t10.24829216\r\n",
      "\"AOKF1022_b2_cP00261\"\t9.176383135\t9.593907737\t9.295004546\t9.083864827\t8.951062624\t9.964998975\t10.40601777\t10.19324114\t9.655692726\t10.13990049\t9.371523874\t9.591439582\t8.843677302\t9.7759508\t10.09041876\t9.660260909\t9.463414239\t9.886652894\t9.226673887\t9.163050089\t8.722151761\t8.796867414\t10.22011755\t10.03016472\t10.10384371\t10.02892475\t9.755778182\t9.92331064\t9.309797438\t10.04326013\t9.740724556\t10.40299996\t9.894117186\t9.787762925\t10.06093292\t10.070228\r\n",
      "\"AOKF1022_g2_cP00510\"\t8.395958546\t8.558903972\t8.296500463\t8.598134888\t8.586736643\t8.637472635\t8.572147867\t8.708218004\t9.137868271\t9.245783529\t8.690977262\t8.79015935\t9.43670665\t8.992279525\t9.064405372\t8.737224602\t8.798197745\t8.899863818\t9.349098238\t9.721433572\t9.164262602\t9.186619677\t8.468318119\t8.477550723\t10.33534281\t9.620895543\t8.684038034\t9.166694902\t9.146223613\t9.080184781\t8.585247224\t9.44301061\t8.369094909\t8.988281447\t8.745167478\t8.89256614\r\n",
      "\"AOKF1024_g2_cP00156\"\t8.4035734\t8.7217391\t8.660244268\t8.729167488\t8.510134889\t8.523374168\t8.535713767\t8.464049014\t8.94370413\t9.178739681\t8.596496336\t8.216576333\t9.096495216\t9.07211917\t8.662740371\t8.669506737\t8.546436881\t8.573745887\t8.43222507\t8.202116967\t9.003742356\t8.597229592\t9.031507644\t8.928315303\t9.159138933\t8.775483924\t8.86505694\t8.558518705\t8.748985369\t8.814513968\t8.441888752\t8.716465801\t8.709762542\t8.758451157\t9.08660415\t9.417109324\r\n",
      "\"AOKF1029_g2_cP00533\"\t11.4242768\t10.66446373\t11.04110528\t10.8613349\t11.0310152\t11.42862298\t10.82206382\t10.86149962\t11.57888574\t11.33506135\t10.85886201\t11.44435693\t10.73691783\t11.0938554\t10.95563241\t11.24780839\t9.702197845\t9.958792112\t9.458937699\t9.47530365\t9.613907169\t9.895562333\t9.934895017\t10.00755584\t9.869879679\t9.640479466\t9.83489065\t10.69779578\t10.19029742\t10.42991646\t9.848112765\t9.833655914\t10.71181668\t11.49687869\t11.92852306\t11.49114242\r\n",
      "\"AOKF1031_g2_cP00692\"\t8.370251729\t8.736343107\t8.523365791\t9.083655113\t8.975146784\t8.627887995\t8.305363746\t8.310976911\t8.024980031\t8.26461239\t8.883329242\t8.925134121\t8.37217139\t8.410349953\t8.397921317\t8.203495289\t8.440558208\t8.272664132\t9.731339211\t9.347867792\t8.987346666\t9.108945339\t8.594765257\t8.794289212\t8.410917182\t8.751894701\t8.384397336\t8.403275861\t8.530084159\t8.818029195\t9.103286003\t8.659610484\t8.517251863\t8.651802604\t8.241466404\t8.099694915\r\n",
      "\"AOKF1034_g2_cP00454\"\t9.26752195\t9.082241833\t9.103749443\t8.913870123\t8.347370386\t8.38592923\t8.50560996\t8.324218578\t8.686459468\t8.323393397\t8.769054923\t8.363963045\t9.235548306\t9.472856194\t9.145113149\t9.450107026\t8.486760919\t9.000680715\t9.049306963\t9.179473772\t8.673074688\t8.546055469\t8.324543583\t8.792090509\t9.056831463\t8.620388431\t9.06345402\t9.077064247\t8.816549359\t8.685427105\t9.393420016\t9.220390544\t8.993108187\t8.573118646\t8.094461193\t8.676602309\r\n",
      "\"AOKF1040_g2_cP00558\"\t9.596890609\t8.556935278\t9.245276711\t9.546611647\t8.426491989\t8.605557421\t9.945608894\t10.29081202\t8.896280276\t8.656590075\t8.959403703\t8.717937342\t8.362310693\t8.275425646\t8.203573428\t8.55123222\t8.249538863\t8.51982006\t8.619265389\t8.536400088\t8.979161578\t8.731089548\t8.636715038\t8.780551909\t8.341767908\t8.552032403\t8.451857498\t8.629559636\t8.7902673\t8.888783749\t8.46038036\t8.594346403\t8.691448653\t8.947153292\t7.880138457\t8.272705725\r\n",
      "\"AOKF1045_g2_cP00610\"\t9.284388762\t9.258621453\t10.04213477\t9.577895365\t9.779106504\t9.680812257\t9.002495678\t9.347329198\t9.936854364\t9.820416141\t9.121961155\t9.254721656\t9.316999633\t9.498593672\t9.350486008\t9.503243324\t9.166116542\t10.03146911\t10.25101115\t10.24177436\t13.63439225\t15.29115959\t10.39898449\t10.10714489\t8.974635374\t10.60523744\t10.60457339\t9.715962929\t9.035667014\t10.01264101\t9.948129891\t10.21467796\t9.403003788\t9.426311702\t9.449679476\t9.558609792\r\n",
      "\"AOKF1046_g2_cP00633\"\t9.313943532\t9.094493156\t9.834212726\t9.693981299\t10.54713263\t10.14895767\t9.192448973\t9.15172914\t9.69429677\t9.442468892\t9.065188026\t9.071677991\t9.401540084\t9.24608613\t8.84386385\t9.032963768\t9.271997762\t9.533009818\t9.341082938\t9.170529968\t9.524517568\t9.553106232\t10.07641078\t9.752859221\t9.845983104\t9.997492702\t10.27021962\t9.947749582\t9.944769501\t10.08866605\t9.90895121\t10.06390577\t8.978624699\t9.023689643\t9.661066803\t9.808833041\r\n"
     ]
    }
   ],
   "source": [
    "#Length of header was determined above. It was found to be 68 lines long. The following code cuts out this header.\n",
    "!sed -e '1,68d' GSE36983_series_matrix.txt > Apalm_expression\n",
    "!head Apalm_expression"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\"AOKF1013_g2_cP00480\"\r\n",
      "\"AOKF1022_b2_cP00261\"\r\n",
      "\"AOKF1022_g2_cP00510\"\r\n",
      "\"AOKF1024_g2_cP00156\"\r\n",
      "\"AOKF1029_g2_cP00533\"\r\n",
      "\"AOKF1031_g2_cP00692\"\r\n",
      "\"AOKF1034_g2_cP00454\"\r\n",
      "\"AOKF1040_g2_cP00558\"\r\n",
      "\"AOKF1045_g2_cP00610\"\r\n",
      "\"AOKF1046_g2_cP00633\"\r\n"
     ]
    }
   ],
   "source": [
    "#Now editing contig IDs\n",
    "!awk '{print $1}' Apalm_expression > Apalm_exp_IDs\n",
    "!head Apalm_exp_IDs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AOKF1013_g2_cP00480\"\r\n",
      "AOKF1022_b2_cP00261\"\r\n",
      "AOKF1022_g2_cP00510\"\r\n",
      "AOKF1024_g2_cP00156\"\r\n",
      "AOKF1029_g2_cP00533\"\r\n",
      "AOKF1031_g2_cP00692\"\r\n",
      "AOKF1034_g2_cP00454\"\r\n",
      "AOKF1040_g2_cP00558\"\r\n",
      "AOKF1045_g2_cP00610\"\r\n",
      "AOKF1046_g2_cP00633\"\r\n"
     ]
    }
   ],
   "source": [
    "!sed 's/[\"]//' Apalm_exp_IDs > Apalm_exp_IDs2\n",
    "!head Apalm_exp_IDs2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AOKF1013_g2_c\r\n",
      "AOKF1022_b2_c\r\n",
      "AOKF1022_g2_c\r\n",
      "AOKF1024_g2_c\r\n",
      "AOKF1029_g2_c\r\n",
      "AOKF1031_g2_c\r\n",
      "AOKF1034_g2_c\r\n",
      "AOKF1040_g2_c\r\n",
      "AOKF1045_g2_c\r\n",
      "AOKF1046_g2_c\r\n"
     ]
    }
   ],
   "source": [
    "!sed 's/P.*//' Apalm_exp_IDs2 > Apalm_exp_IDs3\n",
    "!head Apalm_exp_IDs3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RANDOM00002411\"\r\n",
      "RANDOM00002412\"\r\n",
      "RANDOM00002413\"\r\n",
      "RANDOM00002414\"\r\n",
      "RANDOM00002415\"\r\n",
      "RANDOM00002416\"\r\n",
      "RANDOM00002417\"\r\n",
      "RANDOM00002418\"\r\n",
      "RANDOM00002419\"\r\n",
      "!series_matrix_table_end\r\n"
     ]
    }
   ],
   "source": [
    "!tail Apalm_exp_IDs3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AOKF1013_g2_c\t\"AOKF1013_g2_cP00480\"\t8.752858025\t8.505150716\t9.472343652\t9.408601785\t9.482842255\t9.476583784\t9.927173819\t10.12591784\t10.21694941\t10.2803595\t9.670467134\t10.39275672\t8.7638457\t9.368024881\t8.716463095\t9.018597894\t9.823742505\t9.993787246\t9.133042104\t9.136289757\t9.611697945\t9.356880249\t9.649270508\t9.34780996\t9.496325421\t9.288494465\t9.988583534\t9.754569643\t9.510846045\t9.467298681\t9.366147084\t9.308251806\t9.317440431\t9.143020676\t9.7804244\t10.24829216\r\n",
      "AOKF1022_b2_c\t\"AOKF1022_b2_cP00261\"\t9.176383135\t9.593907737\t9.295004546\t9.083864827\t8.951062624\t9.964998975\t10.40601777\t10.19324114\t9.655692726\t10.13990049\t9.371523874\t9.591439582\t8.843677302\t9.7759508\t10.09041876\t9.660260909\t9.463414239\t9.886652894\t9.226673887\t9.163050089\t8.722151761\t8.796867414\t10.22011755\t10.03016472\t10.10384371\t10.02892475\t9.755778182\t9.92331064\t9.309797438\t10.04326013\t9.740724556\t10.40299996\t9.894117186\t9.787762925\t10.06093292\t10.070228\r\n",
      "AOKF1022_g2_c\t\"AOKF1022_g2_cP00510\"\t8.395958546\t8.558903972\t8.296500463\t8.598134888\t8.586736643\t8.637472635\t8.572147867\t8.708218004\t9.137868271\t9.245783529\t8.690977262\t8.79015935\t9.43670665\t8.992279525\t9.064405372\t8.737224602\t8.798197745\t8.899863818\t9.349098238\t9.721433572\t9.164262602\t9.186619677\t8.468318119\t8.477550723\t10.33534281\t9.620895543\t8.684038034\t9.166694902\t9.146223613\t9.080184781\t8.585247224\t9.44301061\t8.369094909\t8.988281447\t8.745167478\t8.89256614\r\n",
      "AOKF1024_g2_c\t\"AOKF1024_g2_cP00156\"\t8.4035734\t8.7217391\t8.660244268\t8.729167488\t8.510134889\t8.523374168\t8.535713767\t8.464049014\t8.94370413\t9.178739681\t8.596496336\t8.216576333\t9.096495216\t9.07211917\t8.662740371\t8.669506737\t8.546436881\t8.573745887\t8.43222507\t8.202116967\t9.003742356\t8.597229592\t9.031507644\t8.928315303\t9.159138933\t8.775483924\t8.86505694\t8.558518705\t8.748985369\t8.814513968\t8.441888752\t8.716465801\t8.709762542\t8.758451157\t9.08660415\t9.417109324\r\n",
      "AOKF1029_g2_c\t\"AOKF1029_g2_cP00533\"\t11.4242768\t10.66446373\t11.04110528\t10.8613349\t11.0310152\t11.42862298\t10.82206382\t10.86149962\t11.57888574\t11.33506135\t10.85886201\t11.44435693\t10.73691783\t11.0938554\t10.95563241\t11.24780839\t9.702197845\t9.958792112\t9.458937699\t9.47530365\t9.613907169\t9.895562333\t9.934895017\t10.00755584\t9.869879679\t9.640479466\t9.83489065\t10.69779578\t10.19029742\t10.42991646\t9.848112765\t9.833655914\t10.71181668\t11.49687869\t11.92852306\t11.49114242\r\n",
      "AOKF1031_g2_c\t\"AOKF1031_g2_cP00692\"\t8.370251729\t8.736343107\t8.523365791\t9.083655113\t8.975146784\t8.627887995\t8.305363746\t8.310976911\t8.024980031\t8.26461239\t8.883329242\t8.925134121\t8.37217139\t8.410349953\t8.397921317\t8.203495289\t8.440558208\t8.272664132\t9.731339211\t9.347867792\t8.987346666\t9.108945339\t8.594765257\t8.794289212\t8.410917182\t8.751894701\t8.384397336\t8.403275861\t8.530084159\t8.818029195\t9.103286003\t8.659610484\t8.517251863\t8.651802604\t8.241466404\t8.099694915\r\n",
      "AOKF1034_g2_c\t\"AOKF1034_g2_cP00454\"\t9.26752195\t9.082241833\t9.103749443\t8.913870123\t8.347370386\t8.38592923\t8.50560996\t8.324218578\t8.686459468\t8.323393397\t8.769054923\t8.363963045\t9.235548306\t9.472856194\t9.145113149\t9.450107026\t8.486760919\t9.000680715\t9.049306963\t9.179473772\t8.673074688\t8.546055469\t8.324543583\t8.792090509\t9.056831463\t8.620388431\t9.06345402\t9.077064247\t8.816549359\t8.685427105\t9.393420016\t9.220390544\t8.993108187\t8.573118646\t8.094461193\t8.676602309\r\n",
      "AOKF1040_g2_c\t\"AOKF1040_g2_cP00558\"\t9.596890609\t8.556935278\t9.245276711\t9.546611647\t8.426491989\t8.605557421\t9.945608894\t10.29081202\t8.896280276\t8.656590075\t8.959403703\t8.717937342\t8.362310693\t8.275425646\t8.203573428\t8.55123222\t8.249538863\t8.51982006\t8.619265389\t8.536400088\t8.979161578\t8.731089548\t8.636715038\t8.780551909\t8.341767908\t8.552032403\t8.451857498\t8.629559636\t8.7902673\t8.888783749\t8.46038036\t8.594346403\t8.691448653\t8.947153292\t7.880138457\t8.272705725\r\n",
      "AOKF1045_g2_c\t\"AOKF1045_g2_cP00610\"\t9.284388762\t9.258621453\t10.04213477\t9.577895365\t9.779106504\t9.680812257\t9.002495678\t9.347329198\t9.936854364\t9.820416141\t9.121961155\t9.254721656\t9.316999633\t9.498593672\t9.350486008\t9.503243324\t9.166116542\t10.03146911\t10.25101115\t10.24177436\t13.63439225\t15.29115959\t10.39898449\t10.10714489\t8.974635374\t10.60523744\t10.60457339\t9.715962929\t9.035667014\t10.01264101\t9.948129891\t10.21467796\t9.403003788\t9.426311702\t9.449679476\t9.558609792\r\n",
      "AOKF1046_g2_c\t\"AOKF1046_g2_cP00633\"\t9.313943532\t9.094493156\t9.834212726\t9.693981299\t10.54713263\t10.14895767\t9.192448973\t9.15172914\t9.69429677\t9.442468892\t9.065188026\t9.071677991\t9.401540084\t9.24608613\t8.84386385\t9.032963768\t9.271997762\t9.533009818\t9.341082938\t9.170529968\t9.524517568\t9.553106232\t10.07641078\t9.752859221\t9.845983104\t9.997492702\t10.27021962\t9.947749582\t9.944769501\t10.08866605\t9.90895121\t10.06390577\t8.978624699\t9.023689643\t9.661066803\t9.808833041\r\n"
     ]
    }
   ],
   "source": [
    "!paste Apalm_exp_IDs3 Apalm_expression > Apalm_expression2\n",
    "!head Apalm_expression2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AOKF1013_g2_c\t8.752858025\t8.505150716\t9.472343652\t9.408601785\t9.482842255\t9.476583784\t9.927173819\t10.12591784\t10.21694941\t10.2803595\t9.670467134\t10.39275672\t8.7638457\t9.368024881\t8.716463095\t9.018597894\t9.823742505\t9.993787246\t9.133042104\t9.136289757\t9.611697945\t9.356880249\t9.649270508\t9.34780996\t9.496325421\t9.288494465\t9.988583534\t9.754569643\t9.510846045\t9.467298681\t9.366147084\t9.308251806\t9.317440431\t9.143020676\t9.7804244\t10.24829216\r\n",
      "AOKF1022_b2_c\t9.176383135\t9.593907737\t9.295004546\t9.083864827\t8.951062624\t9.964998975\t10.40601777\t10.19324114\t9.655692726\t10.13990049\t9.371523874\t9.591439582\t8.843677302\t9.7759508\t10.09041876\t9.660260909\t9.463414239\t9.886652894\t9.226673887\t9.163050089\t8.722151761\t8.796867414\t10.22011755\t10.03016472\t10.10384371\t10.02892475\t9.755778182\t9.92331064\t9.309797438\t10.04326013\t9.740724556\t10.40299996\t9.894117186\t9.787762925\t10.06093292\t10.070228\r\n",
      "AOKF1022_g2_c\t8.395958546\t8.558903972\t8.296500463\t8.598134888\t8.586736643\t8.637472635\t8.572147867\t8.708218004\t9.137868271\t9.245783529\t8.690977262\t8.79015935\t9.43670665\t8.992279525\t9.064405372\t8.737224602\t8.798197745\t8.899863818\t9.349098238\t9.721433572\t9.164262602\t9.186619677\t8.468318119\t8.477550723\t10.33534281\t9.620895543\t8.684038034\t9.166694902\t9.146223613\t9.080184781\t8.585247224\t9.44301061\t8.369094909\t8.988281447\t8.745167478\t8.89256614\r\n",
      "AOKF1024_g2_c\t8.4035734\t8.7217391\t8.660244268\t8.729167488\t8.510134889\t8.523374168\t8.535713767\t8.464049014\t8.94370413\t9.178739681\t8.596496336\t8.216576333\t9.096495216\t9.07211917\t8.662740371\t8.669506737\t8.546436881\t8.573745887\t8.43222507\t8.202116967\t9.003742356\t8.597229592\t9.031507644\t8.928315303\t9.159138933\t8.775483924\t8.86505694\t8.558518705\t8.748985369\t8.814513968\t8.441888752\t8.716465801\t8.709762542\t8.758451157\t9.08660415\t9.417109324\r\n",
      "AOKF1029_g2_c\t11.4242768\t10.66446373\t11.04110528\t10.8613349\t11.0310152\t11.42862298\t10.82206382\t10.86149962\t11.57888574\t11.33506135\t10.85886201\t11.44435693\t10.73691783\t11.0938554\t10.95563241\t11.24780839\t9.702197845\t9.958792112\t9.458937699\t9.47530365\t9.613907169\t9.895562333\t9.934895017\t10.00755584\t9.869879679\t9.640479466\t9.83489065\t10.69779578\t10.19029742\t10.42991646\t9.848112765\t9.833655914\t10.71181668\t11.49687869\t11.92852306\t11.49114242\r\n",
      "AOKF1031_g2_c\t8.370251729\t8.736343107\t8.523365791\t9.083655113\t8.975146784\t8.627887995\t8.305363746\t8.310976911\t8.024980031\t8.26461239\t8.883329242\t8.925134121\t8.37217139\t8.410349953\t8.397921317\t8.203495289\t8.440558208\t8.272664132\t9.731339211\t9.347867792\t8.987346666\t9.108945339\t8.594765257\t8.794289212\t8.410917182\t8.751894701\t8.384397336\t8.403275861\t8.530084159\t8.818029195\t9.103286003\t8.659610484\t8.517251863\t8.651802604\t8.241466404\t8.099694915\r\n",
      "AOKF1034_g2_c\t9.26752195\t9.082241833\t9.103749443\t8.913870123\t8.347370386\t8.38592923\t8.50560996\t8.324218578\t8.686459468\t8.323393397\t8.769054923\t8.363963045\t9.235548306\t9.472856194\t9.145113149\t9.450107026\t8.486760919\t9.000680715\t9.049306963\t9.179473772\t8.673074688\t8.546055469\t8.324543583\t8.792090509\t9.056831463\t8.620388431\t9.06345402\t9.077064247\t8.816549359\t8.685427105\t9.393420016\t9.220390544\t8.993108187\t8.573118646\t8.094461193\t8.676602309\r\n",
      "AOKF1040_g2_c\t9.596890609\t8.556935278\t9.245276711\t9.546611647\t8.426491989\t8.605557421\t9.945608894\t10.29081202\t8.896280276\t8.656590075\t8.959403703\t8.717937342\t8.362310693\t8.275425646\t8.203573428\t8.55123222\t8.249538863\t8.51982006\t8.619265389\t8.536400088\t8.979161578\t8.731089548\t8.636715038\t8.780551909\t8.341767908\t8.552032403\t8.451857498\t8.629559636\t8.7902673\t8.888783749\t8.46038036\t8.594346403\t8.691448653\t8.947153292\t7.880138457\t8.272705725\r\n",
      "AOKF1045_g2_c\t9.284388762\t9.258621453\t10.04213477\t9.577895365\t9.779106504\t9.680812257\t9.002495678\t9.347329198\t9.936854364\t9.820416141\t9.121961155\t9.254721656\t9.316999633\t9.498593672\t9.350486008\t9.503243324\t9.166116542\t10.03146911\t10.25101115\t10.24177436\t13.63439225\t15.29115959\t10.39898449\t10.10714489\t8.974635374\t10.60523744\t10.60457339\t9.715962929\t9.035667014\t10.01264101\t9.948129891\t10.21467796\t9.403003788\t9.426311702\t9.449679476\t9.558609792\r\n",
      "AOKF1046_g2_c\t9.313943532\t9.094493156\t9.834212726\t9.693981299\t10.54713263\t10.14895767\t9.192448973\t9.15172914\t9.69429677\t9.442468892\t9.065188026\t9.071677991\t9.401540084\t9.24608613\t8.84386385\t9.032963768\t9.271997762\t9.533009818\t9.341082938\t9.170529968\t9.524517568\t9.553106232\t10.07641078\t9.752859221\t9.845983104\t9.997492702\t10.27021962\t9.947749582\t9.944769501\t10.08866605\t9.90895121\t10.06390577\t8.978624699\t9.023689643\t9.661066803\t9.808833041\r\n"
     ]
    }
   ],
   "source": [
    "!cut -f1,1,3- Apalm_expression2 > Apalm_expression\n",
    "!head Apalm_expression"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "!series_matrix_table_end\r\n",
      "AOKF1013_g2_c\t8.752858025\t8.505150716\t9.472343652\t9.408601785\t9.482842255\t9.476583784\t9.927173819\t10.12591784\t10.21694941\t10.2803595\t9.670467134\t10.39275672\t8.7638457\t9.368024881\t8.716463095\t9.018597894\t9.823742505\t9.993787246\t9.133042104\t9.136289757\t9.611697945\t9.356880249\t9.649270508\t9.34780996\t9.496325421\t9.288494465\t9.988583534\t9.754569643\t9.510846045\t9.467298681\t9.366147084\t9.308251806\t9.317440431\t9.143020676\t9.7804244\t10.24829216\r\n",
      "AOKF1022_b2_c\t9.176383135\t9.593907737\t9.295004546\t9.083864827\t8.951062624\t9.964998975\t10.40601777\t10.19324114\t9.655692726\t10.13990049\t9.371523874\t9.591439582\t8.843677302\t9.7759508\t10.09041876\t9.660260909\t9.463414239\t9.886652894\t9.226673887\t9.163050089\t8.722151761\t8.796867414\t10.22011755\t10.03016472\t10.10384371\t10.02892475\t9.755778182\t9.92331064\t9.309797438\t10.04326013\t9.740724556\t10.40299996\t9.894117186\t9.787762925\t10.06093292\t10.070228\r\n",
      "AOKF1022_g2_c\t8.395958546\t8.558903972\t8.296500463\t8.598134888\t8.586736643\t8.637472635\t8.572147867\t8.708218004\t9.137868271\t9.245783529\t8.690977262\t8.79015935\t9.43670665\t8.992279525\t9.064405372\t8.737224602\t8.798197745\t8.899863818\t9.349098238\t9.721433572\t9.164262602\t9.186619677\t8.468318119\t8.477550723\t10.33534281\t9.620895543\t8.684038034\t9.166694902\t9.146223613\t9.080184781\t8.585247224\t9.44301061\t8.369094909\t8.988281447\t8.745167478\t8.89256614\r\n",
      "AOKF1024_g2_c\t8.4035734\t8.7217391\t8.660244268\t8.729167488\t8.510134889\t8.523374168\t8.535713767\t8.464049014\t8.94370413\t9.178739681\t8.596496336\t8.216576333\t9.096495216\t9.07211917\t8.662740371\t8.669506737\t8.546436881\t8.573745887\t8.43222507\t8.202116967\t9.003742356\t8.597229592\t9.031507644\t8.928315303\t9.159138933\t8.775483924\t8.86505694\t8.558518705\t8.748985369\t8.814513968\t8.441888752\t8.716465801\t8.709762542\t8.758451157\t9.08660415\t9.417109324\r\n",
      "AOKF1029_g2_c\t11.4242768\t10.66446373\t11.04110528\t10.8613349\t11.0310152\t11.42862298\t10.82206382\t10.86149962\t11.57888574\t11.33506135\t10.85886201\t11.44435693\t10.73691783\t11.0938554\t10.95563241\t11.24780839\t9.702197845\t9.958792112\t9.458937699\t9.47530365\t9.613907169\t9.895562333\t9.934895017\t10.00755584\t9.869879679\t9.640479466\t9.83489065\t10.69779578\t10.19029742\t10.42991646\t9.848112765\t9.833655914\t10.71181668\t11.49687869\t11.92852306\t11.49114242\r\n",
      "AOKF1031_g2_c\t8.370251729\t8.736343107\t8.523365791\t9.083655113\t8.975146784\t8.627887995\t8.305363746\t8.310976911\t8.024980031\t8.26461239\t8.883329242\t8.925134121\t8.37217139\t8.410349953\t8.397921317\t8.203495289\t8.440558208\t8.272664132\t9.731339211\t9.347867792\t8.987346666\t9.108945339\t8.594765257\t8.794289212\t8.410917182\t8.751894701\t8.384397336\t8.403275861\t8.530084159\t8.818029195\t9.103286003\t8.659610484\t8.517251863\t8.651802604\t8.241466404\t8.099694915\r\n",
      "AOKF1034_g2_c\t9.26752195\t9.082241833\t9.103749443\t8.913870123\t8.347370386\t8.38592923\t8.50560996\t8.324218578\t8.686459468\t8.323393397\t8.769054923\t8.363963045\t9.235548306\t9.472856194\t9.145113149\t9.450107026\t8.486760919\t9.000680715\t9.049306963\t9.179473772\t8.673074688\t8.546055469\t8.324543583\t8.792090509\t9.056831463\t8.620388431\t9.06345402\t9.077064247\t8.816549359\t8.685427105\t9.393420016\t9.220390544\t8.993108187\t8.573118646\t8.094461193\t8.676602309\r\n",
      "AOKF1040_g2_c\t9.596890609\t8.556935278\t9.245276711\t9.546611647\t8.426491989\t8.605557421\t9.945608894\t10.29081202\t8.896280276\t8.656590075\t8.959403703\t8.717937342\t8.362310693\t8.275425646\t8.203573428\t8.55123222\t8.249538863\t8.51982006\t8.619265389\t8.536400088\t8.979161578\t8.731089548\t8.636715038\t8.780551909\t8.341767908\t8.552032403\t8.451857498\t8.629559636\t8.7902673\t8.888783749\t8.46038036\t8.594346403\t8.691448653\t8.947153292\t7.880138457\t8.272705725\r\n",
      "AOKF1045_g2_c\t9.284388762\t9.258621453\t10.04213477\t9.577895365\t9.779106504\t9.680812257\t9.002495678\t9.347329198\t9.936854364\t9.820416141\t9.121961155\t9.254721656\t9.316999633\t9.498593672\t9.350486008\t9.503243324\t9.166116542\t10.03146911\t10.25101115\t10.24177436\t13.63439225\t15.29115959\t10.39898449\t10.10714489\t8.974635374\t10.60523744\t10.60457339\t9.715962929\t9.035667014\t10.01264101\t9.948129891\t10.21467796\t9.403003788\t9.426311702\t9.449679476\t9.558609792\r\n"
     ]
    }
   ],
   "source": [
    "!sort Apalm_expression > Apalm_expression_sorted\n",
    "!head Apalm_expression_sorted"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AOKF1013_g2_c 8.752858025 8.505150716 9.472343652 9.408601785 9.482842255 9.476583784 9.927173819 10.12591784 10.21694941 10.2803595 9.670467134 10.39275672 8.7638457 9.368024881 8.716463095 9.018597894 9.823742505 9.993787246 9.133042104 9.136289757 9.611697945 9.356880249 9.649270508 9.34780996 9.496325421 9.288494465 9.988583534 9.754569643 9.510846045 9.467298681 9.366147084 9.308251806 9.317440431 9.143020676 9.7804244 10.24829216 0.261194\r\n",
      "AOKF1022_b2_c 9.176383135 9.593907737 9.295004546 9.083864827 8.951062624 9.964998975 10.40601777 10.19324114 9.655692726 10.13990049 9.371523874 9.591439582 8.843677302 9.7759508 10.09041876 9.660260909 9.463414239 9.886652894 9.226673887 9.163050089 8.722151761 8.796867414 10.22011755 10.03016472 10.10384371 10.02892475 9.755778182 9.92331064 9.309797438 10.04326013 9.740724556 10.40299996 9.894117186 9.787762925 10.06093292 10.070228 1.21084\r\n",
      "AOKF1022_g2_c 8.395958546 8.558903972 8.296500463 8.598134888 8.586736643 8.637472635 8.572147867 8.708218004 9.137868271 9.245783529 8.690977262 8.79015935 9.43670665 8.992279525 9.064405372 8.737224602 8.798197745 8.899863818 9.349098238 9.721433572 9.164262602 9.186619677 8.468318119 8.477550723 10.33534281 9.620895543 8.684038034 9.166694902 9.146223613 9.080184781 8.585247224 9.44301061 8.369094909 8.988281447 8.745167478 8.89256614 0.933676\r\n",
      "AOKF1024_g2_c 8.4035734 8.7217391 8.660244268 8.729167488 8.510134889 8.523374168 8.535713767 8.464049014 8.94370413 9.178739681 8.596496336 8.216576333 9.096495216 9.07211917 8.662740371 8.669506737 8.546436881 8.573745887 8.43222507 8.202116967 9.003742356 8.597229592 9.031507644 8.928315303 9.159138933 8.775483924 8.86505694 8.558518705 8.748985369 8.814513968 8.441888752 8.716465801 8.709762542 8.758451157 9.08660415 9.417109324 0.46793\r\n",
      "AOKF1029_g2_c 11.4242768 10.66446373 11.04110528 10.8613349 11.0310152 11.42862298 10.82206382 10.86149962 11.57888574 11.33506135 10.85886201 11.44435693 10.73691783 11.0938554 10.95563241 11.24780839 9.702197845 9.958792112 9.458937699 9.47530365 9.613907169 9.895562333 9.934895017 10.00755584 9.869879679 9.640479466 9.83489065 10.69779578 10.19029742 10.42991646 9.848112765 9.833655914 10.71181668 11.49687869 11.92852306 11.49114242 0.305319\r\n",
      "AOKF1031_g2_c 8.370251729 8.736343107 8.523365791 9.083655113 8.975146784 8.627887995 8.305363746 8.310976911 8.024980031 8.26461239 8.883329242 8.925134121 8.37217139 8.410349953 8.397921317 8.203495289 8.440558208 8.272664132 9.731339211 9.347867792 8.987346666 9.108945339 8.594765257 8.794289212 8.410917182 8.751894701 8.384397336 8.403275861 8.530084159 8.818029195 9.103286003 8.659610484 8.517251863 8.651802604 8.241466404 8.099694915 0.476647\r\n",
      "AOKF1034_g2_c 9.26752195 9.082241833 9.103749443 8.913870123 8.347370386 8.38592923 8.50560996 8.324218578 8.686459468 8.323393397 8.769054923 8.363963045 9.235548306 9.472856194 9.145113149 9.450107026 8.486760919 9.000680715 9.049306963 9.179473772 8.673074688 8.546055469 8.324543583 8.792090509 9.056831463 8.620388431 9.06345402 9.077064247 8.816549359 8.685427105 9.393420016 9.220390544 8.993108187 8.573118646 8.094461193 8.676602309 0.250371\r\n",
      "AOKF1040_g2_c 9.596890609 8.556935278 9.245276711 9.546611647 8.426491989 8.605557421 9.945608894 10.29081202 8.896280276 8.656590075 8.959403703 8.717937342 8.362310693 8.275425646 8.203573428 8.55123222 8.249538863 8.51982006 8.619265389 8.536400088 8.979161578 8.731089548 8.636715038 8.780551909 8.341767908 8.552032403 8.451857498 8.629559636 8.7902673 8.888783749 8.46038036 8.594346403 8.691448653 8.947153292 7.880138457 8.272705725 1.11148\r\n",
      "AOKF1045_g2_c 9.284388762 9.258621453 10.04213477 9.577895365 9.779106504 9.680812257 9.002495678 9.347329198 9.936854364 9.820416141 9.121961155 9.254721656 9.316999633 9.498593672 9.350486008 9.503243324 9.166116542 10.03146911 10.25101115 10.24177436 13.63439225 15.29115959 10.39898449 10.10714489 8.974635374 10.60523744 10.60457339 9.715962929 9.035667014 10.01264101 9.948129891 10.21467796 9.403003788 9.426311702 9.449679476 9.558609792 0.415524\r\n",
      "AOKF1046_g2_c 9.313943532 9.094493156 9.834212726 9.693981299 10.54713263 10.14895767 9.192448973 9.15172914 9.69429677 9.442468892 9.065188026 9.071677991 9.401540084 9.24608613 8.84386385 9.032963768 9.271997762 9.533009818 9.341082938 9.170529968 9.524517568 9.553106232 10.07641078 9.752859221 9.845983104 9.997492702 10.27021962 9.947749582 9.944769501 10.08866605 9.90895121 10.06390577 8.978624699 9.023689643 9.661066803 9.808833041 0.278746\r\n"
     ]
    }
   ],
   "source": [
    "!join Apalm_expression_sorted ID_CpG.sorted > Apalm_exp_CpG\n",
    "!head Apalm_exp_CpG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AOKF1013_g2_c\t8.752858025\t8.505150716\t9.472343652\t9.408601785\t9.482842255\t9.476583784\t9.927173819\t10.12591784\t10.21694941\t10.2803595\t9.670467134\t10.39275672\t8.7638457\t9.368024881\t8.716463095\t9.018597894\t9.823742505\t9.993787246\t9.133042104\t9.136289757\t9.611697945\t9.356880249\t9.649270508\t9.34780996\t9.496325421\t9.288494465\t9.988583534\t9.754569643\t9.510846045\t9.467298681\t9.366147084\t9.308251806\t9.317440431\t9.143020676\t9.7804244\t10.24829216\t0.261194\r\n",
      "AOKF1022_b2_c\t9.176383135\t9.593907737\t9.295004546\t9.083864827\t8.951062624\t9.964998975\t10.40601777\t10.19324114\t9.655692726\t10.13990049\t9.371523874\t9.591439582\t8.843677302\t9.7759508\t10.09041876\t9.660260909\t9.463414239\t9.886652894\t9.226673887\t9.163050089\t8.722151761\t8.796867414\t10.22011755\t10.03016472\t10.10384371\t10.02892475\t9.755778182\t9.92331064\t9.309797438\t10.04326013\t9.740724556\t10.40299996\t9.894117186\t9.787762925\t10.06093292\t10.070228\t1.21084\r\n",
      "AOKF1022_g2_c\t8.395958546\t8.558903972\t8.296500463\t8.598134888\t8.586736643\t8.637472635\t8.572147867\t8.708218004\t9.137868271\t9.245783529\t8.690977262\t8.79015935\t9.43670665\t8.992279525\t9.064405372\t8.737224602\t8.798197745\t8.899863818\t9.349098238\t9.721433572\t9.164262602\t9.186619677\t8.468318119\t8.477550723\t10.33534281\t9.620895543\t8.684038034\t9.166694902\t9.146223613\t9.080184781\t8.585247224\t9.44301061\t8.369094909\t8.988281447\t8.745167478\t8.89256614\t0.933676\r\n",
      "AOKF1024_g2_c\t8.4035734\t8.7217391\t8.660244268\t8.729167488\t8.510134889\t8.523374168\t8.535713767\t8.464049014\t8.94370413\t9.178739681\t8.596496336\t8.216576333\t9.096495216\t9.07211917\t8.662740371\t8.669506737\t8.546436881\t8.573745887\t8.43222507\t8.202116967\t9.003742356\t8.597229592\t9.031507644\t8.928315303\t9.159138933\t8.775483924\t8.86505694\t8.558518705\t8.748985369\t8.814513968\t8.441888752\t8.716465801\t8.709762542\t8.758451157\t9.08660415\t9.417109324\t0.46793\r\n",
      "AOKF1029_g2_c\t11.4242768\t10.66446373\t11.04110528\t10.8613349\t11.0310152\t11.42862298\t10.82206382\t10.86149962\t11.57888574\t11.33506135\t10.85886201\t11.44435693\t10.73691783\t11.0938554\t10.95563241\t11.24780839\t9.702197845\t9.958792112\t9.458937699\t9.47530365\t9.613907169\t9.895562333\t9.934895017\t10.00755584\t9.869879679\t9.640479466\t9.83489065\t10.69779578\t10.19029742\t10.42991646\t9.848112765\t9.833655914\t10.71181668\t11.49687869\t11.92852306\t11.49114242\t0.305319\r\n",
      "AOKF1031_g2_c\t8.370251729\t8.736343107\t8.523365791\t9.083655113\t8.975146784\t8.627887995\t8.305363746\t8.310976911\t8.024980031\t8.26461239\t8.883329242\t8.925134121\t8.37217139\t8.410349953\t8.397921317\t8.203495289\t8.440558208\t8.272664132\t9.731339211\t9.347867792\t8.987346666\t9.108945339\t8.594765257\t8.794289212\t8.410917182\t8.751894701\t8.384397336\t8.403275861\t8.530084159\t8.818029195\t9.103286003\t8.659610484\t8.517251863\t8.651802604\t8.241466404\t8.099694915\t0.476647\r\n",
      "AOKF1034_g2_c\t9.26752195\t9.082241833\t9.103749443\t8.913870123\t8.347370386\t8.38592923\t8.50560996\t8.324218578\t8.686459468\t8.323393397\t8.769054923\t8.363963045\t9.235548306\t9.472856194\t9.145113149\t9.450107026\t8.486760919\t9.000680715\t9.049306963\t9.179473772\t8.673074688\t8.546055469\t8.324543583\t8.792090509\t9.056831463\t8.620388431\t9.06345402\t9.077064247\t8.816549359\t8.685427105\t9.393420016\t9.220390544\t8.993108187\t8.573118646\t8.094461193\t8.676602309\t0.250371\r\n",
      "AOKF1040_g2_c\t9.596890609\t8.556935278\t9.245276711\t9.546611647\t8.426491989\t8.605557421\t9.945608894\t10.29081202\t8.896280276\t8.656590075\t8.959403703\t8.717937342\t8.362310693\t8.275425646\t8.203573428\t8.55123222\t8.249538863\t8.51982006\t8.619265389\t8.536400088\t8.979161578\t8.731089548\t8.636715038\t8.780551909\t8.341767908\t8.552032403\t8.451857498\t8.629559636\t8.7902673\t8.888783749\t8.46038036\t8.594346403\t8.691448653\t8.947153292\t7.880138457\t8.272705725\t1.11148\r\n",
      "AOKF1045_g2_c\t9.284388762\t9.258621453\t10.04213477\t9.577895365\t9.779106504\t9.680812257\t9.002495678\t9.347329198\t9.936854364\t9.820416141\t9.121961155\t9.254721656\t9.316999633\t9.498593672\t9.350486008\t9.503243324\t9.166116542\t10.03146911\t10.25101115\t10.24177436\t13.63439225\t15.29115959\t10.39898449\t10.10714489\t8.974635374\t10.60523744\t10.60457339\t9.715962929\t9.035667014\t10.01264101\t9.948129891\t10.21467796\t9.403003788\t9.426311702\t9.449679476\t9.558609792\t0.415524\r\n",
      "AOKF1046_g2_c\t9.313943532\t9.094493156\t9.834212726\t9.693981299\t10.54713263\t10.14895767\t9.192448973\t9.15172914\t9.69429677\t9.442468892\t9.065188026\t9.071677991\t9.401540084\t9.24608613\t8.84386385\t9.032963768\t9.271997762\t9.533009818\t9.341082938\t9.170529968\t9.524517568\t9.553106232\t10.07641078\t9.752859221\t9.845983104\t9.997492702\t10.27021962\t9.947749582\t9.944769501\t10.08866605\t9.90895121\t10.06390577\t8.978624699\t9.023689643\t9.661066803\t9.808833041\t0.278746\r\n"
     ]
    }
   ],
   "source": [
    "!tr ' ' \\\\t < Apalm_exp_CpG > Apalm_exp_CpG2\n",
    "!head Apalm_exp_CpG2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Now moving to R for analysis. See /analyses/scripts/Expression.R"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
